arm-analysisservices/2017-06-09/swagger/analysisservices.json

```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
-          "$ref": "#/definitions/BackupConfiguration"
+        "BackupBlobContainerUri": {
```

Comment:
Why is this not camelCased? 

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/241267885#L1273